### PR TITLE
Support event logs in VM mode

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: assets/js/universal-dapp.js [hunk 1]
```diff
@@ -150,24 +150,33 @@ UniversalDApp.prototype.getInstanceInterface = function (contract, address, $tar
...
         $events = $('<div class="events"/>');
+
+        var parseLogs = function(err,response) {
+            $event = $('<div class="event" />')
+
+            var $close = $('<div class="udapp-close" />')
+            $close.click( function(){ $event.remove(); } )
+
+            $event.append( $('<span class="name"/>').text(response.event) )
+                .append( $('<span class="args" />').text( JSON.stringify(response.args, null, 2) ) )
```
Comment (chriseth) [inline assets/js/universal-dapp.js:161]: Please don't add additional spaces inside `()`.

Comment (axic) [inline assets/js/universal-dapp.js:161]: That is actually how it was (see below), just moved the block to a function. Should it be changed?

(Too bad github can't show alternatively the diff with `diff -w`)

Comment (chriseth) [inline assets/js/universal-dapp.js:161]: Ah sorry. In the long run, we should find some common coding style...

Comment (axic) [inline assets/js/universal-dapp.js:161]: What bugs me is that half of the file has DOS-style new lines (\r\n).

**Pull request merged**